README.md

```diff
@@ -79,9 +79,9 @@ To maintain a pristine source tree, CMake encourages to perform an out-of-source
 
     $ mkdir build && cd build
     $ cmake ..
```

Comment:
nit: Might be explicit:
```suggestion
    $ cmake -S ..
```
?

Reply:
Hm, yeah, I think shorter is better, and omitting the -S is also more common. (See the official tutorial https://cmake.org/cmake/help/latest/guide/tutorial/A%20Basic%20Starting%20Point.html for example.)

Reply:
OK, nm.